src/commons/TestUtils.ts

```diff
@@ -27,4 +30,24 @@ export namespace TestUtils {
   export function extractNumbersFromString(str: string): string {
     return str.replace(/[^0-9]/g, '');
   }
+
+  /**
+   * Return object of data by file path
+   *  path File located in wdio config file,
+   */
+  export function getDataFromFile(): object {
+    // @ts-ignore
+    const dataFilename = browser.config.configDataFilePath;
+    const dataFilePath = path.resolve(process.cwd(), dataFilename);
+    return require(dataFilePath);
+  }
+
+  /**
+   * Get data by tag from e2eTestData file
+   * @param dataTag string
+   */
+  export function getData(dataTag: string = process.env.TEST_DATA_TAG): object {
```

Comment:
TEST_DATA_TAG can be undefined here

Reply:
@FelixZilber if it will be `undefined` we will get `undefined` as a return value
Do you find it problematic?

Reply:
@nirgur better to add some log in this case. in case if something goes wrong, it will be much easier to understand the root cause

Reply:
👍